README.md

```diff
@@ -32,7 +32,7 @@ The module comes with the following commands:
 
     * On Windows, a temporary _batch file_ rather than a direct `cmd.exe /c` call is used behind the scenes, (not just) for technical reasons. This means that batch-file syntax must be used, which notably means that loop variables must use `%%`, not just `%`, and that you may escape `%` as `%%` - arguably, this is for the better anyway. The only caveat is that aborting a long-running command with <kbd>Ctrl-C</kbd> will present the infamous `Terminate batch file (y/n)?` prompt; simple repeat <kbd>Ctrl-C</kbd> to complete the termination.
 
-    * `--` can be used to dismabiguate pass-through arguments from `ins` own parameters; if you need to pass `--` as a pass-through argument, first precede all pass-through arguments with `--`, and then use 
+    * `--` can be used to dismabiguate pass-through arguments from `ins` own parameters; if you need to pass `--` as a pass-through argument, first precede all pass-through arguments with `--` and then use `--` again.
```

Comment:
Thanks for catching that.

Reply:
Given that I think this is the only fix to make, I've just applied it directly, and I can think we can close this PR - thanks for the contribution.